libcontainer/configs/cgroup_unix.go

```diff
@@ -120,5 +120,5 @@ type Resources struct {
 	NetPrioIfpriomap []*IfPrioMap `json:"net_prio_ifpriomap"`
 
 	// Set class identifier for container's network packets
-	NetClsClassid string `json:"net_cls_classid"`
+	NetClsClassid *uint32 `json:"net_cls_classid"`
```

Comment:
We generally don't use pointers for the internal libcontainer configs. Though, I'm not sure what a `NetClsClassid` of `0` means so this might be fine. WDYT @mrunalp